app/service/auth_svc.py

```diff
@@ -118,13 +145,19 @@ def request_has_valid_api_key(self, request):
     async def request_has_valid_user_session(self, request):
         return await aiohttp_security_api.authorized_userid(request) is not None
 
+    async def provide_verified_login_response(self, request, username):
```

Comment:
What do you think about `handle_successful_login()` or something similar as an alternative name? I think it's the `provide` part of the method name that's throwing me. Maybe it's just me but I keep thinking of guice when I read this: https://github.com/google/guice/wiki/ProvidesMethods

Reply:
I like it, will change accordingly